OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1

```diff
@@ -69,7 +69,9 @@ function Get-TargetResource {
     }
     else {
         Write-Verbose "Windows service: Not installed"
-        $existingEnsure = "Absent"
+        if ($existingEnsure -eq 'Present') {
```

Comment:
Not sure I understand this logic. Why set `$existingEnsure` to `Installed` when it's already set to `Present`?
We don't need `$existingEnsure = "Absent"` if that condition isn't met any more?

Reply:
At this point in the code, `$existingEnsure` equals `Present` if the registry key is present. But then we subsequently check if the service is present, and if the service isn't there, that means its installed, not configured.

The fact that you're raising this probably means it needs a refactor.